examples/miscellaneous/range-timeseries.py

```diff
@@ -31,7 +31,7 @@
 __credits__ = 'Alex Urban <alexander.urban@ligo.org>'
 
 # First, we need to load some data. We can `fetch` the
-# `public data <https://www.gw-openscience.org/catalog/>`__
+# `public data <https://gwosc.org/catalog/>`__
```

Comment:
I don't know if it matters, since this is a comment, but neither the original URL (https://www.gw-openscience.org/catalog/) nor the modified URL (https://gwosc.org/catalog/) works.